eslint.config.mjs

```diff
@@ -0,0 +1,145 @@
+import importPlugin from 'eslint-plugin-import-x';
+import react from 'eslint-plugin-react';
+import globals from 'globals';
+import babelParser from '@babel/eslint-parser';
+import path from 'node:path';
+import { fileURLToPath } from 'node:url';
+import js from '@eslint/js';
+import { FlatCompat } from '@eslint/eslintrc';
+
+const __filename = fileURLToPath(import.meta.url);
+const __dirname = path.dirname(__filename);
+const compat = new FlatCompat({
+  baseDirectory: __dirname,
+  recommendedConfig: js.configs.recommended,
+  allConfig: js.configs.all,
+});
+
+export default [
+  {
+    ignores: [
+      '**/.vscode/',
+      '.github/',
+      'media/',
+      'static/',
+      '**/coverage/',
+      'docs/_build/',
+      '**/package-lock.json',
+      '**/specs/',
+      '**/?venv/',
+      'translate/dist/',
+      // Jinja templates
+      'translate/public/translate.html',
+      '**/templates/**/*.html',
+      // Vendored code
+      'error_pages/css/blockrain.css',
+      'error_pages/js/',
+      'pontoon/base/static/css/boilerplate.css',
+      'pontoon/base/static/css/fontawesome-all.css',
+      'pontoon/base/static/css/nprogress.css',
+      'pontoon/base/static/js/lib/',
+    ],
+  },
+  ...compat.extends('eslint:recommended', 'plugin:react/recommended'),
```

Comment:
This should be pretty easy to replace with a proper non-compat solution, something like this:
```suggestion
  js.configs.recommended,
  reactPlugin.configs.flat.recommended,
```

Reply:
Removed (this was generated by the conversion tool).